digid_eherkenning/models/certificates.py

```diff
@@ -147,9 +162,24 @@ class Meta:
                 violation_error_message=_(
                     "This configuration and certificate combination already exists."
                 ),
-            ),
+            )
         ]
 
+    def clean(self):
+        if not (
+            self.certificate.valid_from
+            <= self.activation_date
+            <= self.certificate.expiry_date
```

Comment:
```suggestion
            < self.certificate.expiry_date
```

I would go with less than, because equal to means it immediately expires so it's not useful.